project/MultiNode.scala

```diff
@@ -69,11 +69,14 @@ object MultiNode extends AutoPlugin {
           (executeTests in Test) := {
             val testResults = (executeTests in Test).value
             val multiNodeResults = multiExecuteTests.value
-            val overall =
-              if (testResults.overall.id < multiNodeResults.overall.id)
-                multiNodeResults.overall
-              else
-                testResults.overall
+            //        // FIXME no idea what this was doing
+            //        val overall =
+            //          if (testResults.overall.id < multiNodeResults.overall.id)
+            //            multiNodeResults.overall
+            //          else
+            //            testResults.overall
+            val overall = testResults.overall
```

Comment:
did it break if we wanted to keep it? I believe this makes the tests fail when multinode tests fail so we may need to keep it.

Reply:
Will use this to make it work: https://github.com/akka/akka-http/pull/1435/files#r147072897